README.md

```diff
@@ -402,3 +402,15 @@ Example using cURL:
 ```sh
 curl -X DELETE localhost:5000/services/my-service
 ```
+
+##### Special Chars
+
+If you have any service with special chars like _@_, _/_, etc... you need to use html hex codes. See detailed list [HTML SpecialChars]
```

Comment:
HTML encoding is different than URL encoding. For the import map deployer, it uses URI encoding - not HTML encoding. The correct link to share that documents this is https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent

Please update this to only refer to URI encoding - not HTML encoding.

Reply:
Updated!